tests/models/led/test_tokenization_led.py

```diff
@@ -171,3 +171,16 @@ def test_embeded_special_tokens(self):
                 self.assertSequenceEqual(
                     tokens_r_str, ["<s>", "A", ",", "<mask>", "ĠAllen", "N", "LP", "Ġsentence", ".", "</s>"]
                 )
+
+    @require_torch
+    def test_global_attention_mask(self):
+        for tokenizer, pretrained_name, kwargs in self.tokenizers_list:
+            with self.subTest(f"{tokenizer.__class__.__name__} ({pretrained_name})"):
+                tokenizer_r = self.rust_tokenizer_class.from_pretrained(pretrained_name, **kwargs)
+                src_text = ["A long paragraph.", "Hi I am using huggingface transformers"]
```

Comment:
Let's use `A short text` maybe? It's the shorter one of the two texts :-)
Don't forget to update the necessary mask and expected outputs